### ISSUE TITLE
Display estimated impacts cloud instances data

### ISSUE THREAD

**Issue by demeringo** [Boavizta/boavizta-front#58]
Boavizta API also provides estimations of impacts of AWS cloud instances.

We could add a tab to query and display it (similar to the server tab) ?

**Comment by AirLoren** [comment]
Feature preview available here : https://boavizta-dataviz.cmakers.io/cloudimpact
Comments are welcome.

**Comment by demeringo** [comment]
Thank you @AirLoren for this work (and the reactivity 👍 👍) !

The optional display of the underlying server configuration and usage factors used to assess the instance is very cool.
This level of detail (usage factors) is lacking in current cloud-scanner metrics... and I am thinking of adding it now (https://github.com/Boavizta/cloud-scanner/issues/185) 😉

One thing that come to my mind:
- It could be interesting to display the PE in Wh/ KWh...  _in addition_ to J (imho, it make the value easier to compare to the consumption of a end user device).

What do you think ?

**Comment by AirLoren** [comment]
I agree this information is interesting but I can't find a good way to display it while keeping consistency with other criterias. Where and how would you display this value?
